content/resources/chapter-7/mempool-transaction-1.tsx

```diff
@@ -24,33 +24,41 @@ const javascript = {
     `function assembleBlock(mempool) {
   // This block constructor opportunistically includes transactions with unconfirmed parents.
   // If a transaction has unconfirmed parents, but those parents are all already included in the block, then it is valid for inclusion.
-  const block = [];
-  let block_weight = 0;
-  for (const tx of mempool) {
-      tx.feerate = parseFloat(tx.fee) / parseFloat(tx.weight);
-  }
-  // Construct dictionary for fast lookup
-  const txs = new Map([...mempool.map(tx => [tx.txid, tx])].sort((a, b) => b[1].feerate - a[1].feerate));
+
```

Comment:
Seems overkill to have both JS and Python versions of pseudocode, which is really a third language. The editor colors/highlighting makes it look a bit weird as well. Could it just be a monospace text field?